docs/Release Notes/release-notes-2023.md

```diff
@@ -4,6 +4,50 @@ description: "Paralus release history with release notes for 2023"
 slug: /release-notes/2023
 ---
 
+---
+## v0.2.4
+
+**April 28, 2023**
+
+**paralus/paralus**
+
+**What's Changed**
+
+- Fix the number of wg.add bug - [tyut22316](https://github.com/tyut22316)
+- chore(deps): Bump docker/metadata-action from 4.3.0 to 4.4.0 by @dependabot
```

Comment:
This is not needed in release notes. (This does not add any value to the user)

Reply:
Same is applicable to all below `chore(deps)` entries by dependabot. Let's remove all of them.

Reply:
@akshay196 Agreed, just another thought - current release notes is just a replica of changelogs and there's no added value. Probably we should come up with some better approach for release notes. like for e.g. current release has a feature which can be highlighted. If someone needs to know the complete changelog they can anyhow get it from github. wdyt ?

Reply:
Agree with you, Nirav. Let's do some research on release notes. Then decide how to make it more useful.

Reply:
@techmaharaj @Saim-Safdar @venu-rafay  let us know if you've any suggestions on how release notes should be.

Reply:
Agree as well.
Instead of taking comments from change log,  we manually add release notes. The simple format can be ,
1. List of all major features that were added.
2. List of critical enhancements.
3. List of all but only important bug fixes and/or the issues addressed.
4. Any design change or code restructure that was done part of this release.

 We just include what is applicable for a given release.

Reply:
@akshay196 somehow missed to update this, have removed chore(deps) from release notes as of now.